pipeline.code-workspace

```diff
@@ -1,17 +0,0 @@
-{
-	"folders": [
-		{
-			"path": "."
-		}
-	],
```

Comment:
why is this getting removed?

Reply:
Because we are not using it anymore. Each project contained in pipeline (the worker and the local runner) has its own workspace file now.